File: Src/Codecamp.Feedback/Models/FeedbackSession.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Codecamp.Feedback.Models
{
	public class FeedbackSession
	{
		[JsonIgnore]
		public int Id { get; set; }
		public int SessionId { get; set; }
		[JsonIgnore]
		public Session Session { get; set; }
		[JsonIgnore]
		public Guid FeedbackUserId { get; set; }
		[JsonIgnore]
		public FeedbackUser FeedbackUser { get; set; }
		public int ContentRating { get; set; }
		public int PresentationRating { get; set; }
		[JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
		public string Notes { get; set; }
	}
}

File: Src/App/js/MVVM/Feedback/FeedbackSession.js
$.extend(Codecamp.viewModels, {
	FeedbackSession: function (data) {
		var viewModel = ko.mapping.fromJS({
			ContentRating: null,
			PresentationRating: null,
			Notes: null
		});
		ko.mapping.fromJS(data, viewModel)
		return $.extend(viewModel, {
		});
	}
});